application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -172,27 +172,33 @@
               #end ##
             &lt;/tr&gt;
           #end
+
+          ## third table row
```

Comment:
This comment is not very clear. Maybe is referring to the table footer, which is the 3rd row only if one user votes. Instead it could be documented as `Results row.` or `Table footer`.